starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +19,43 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(arr) {
+  if (!arr.length) {return null}
+  if (arr.length === 1) {return arr[0]}
```

Comment:
No debería ser necesario esta condición si se gestiona bien dentro del `for`, ya que al tener un solo elemento, lo puedes devolver directamente por ser la palabra más larga